az-digital/az_quickstart
# Add view and displays for page by category (closes #573)

This pull request adds the needed view/displays for Page by Category

## Description
This pull request adds the needed view/displays for Page by Category. 
-Added existing fields for photo and summary to the Page content type
-Created summary tab on form display (where you add photo and summary)
-Add new field for page category
-Created categorization tab on form display
-Added view displays for Teaser and Card and applied styling to match QS1
-Created view "Page by Category"
-Updated the jquery selector in the Chosen settings to include select#edit-field-az-page-category

Two different displays exist:
-Row: https://d0311dc6-cf50-4ee7-8723-21866052170e.probo.build/page-list
![image](https://user-images.githubusercontent.com/10873398/116485590-7341d680-a840-11eb-8faf-b5fdfd466922.png)

-Card: https://d0311dc6-cf50-4ee7-8723-21866052170e.probo.build/page-grid
![image](https://user-images.githubusercontent.com/10873398/116485655-9b313a00-a840-11eb-864f-0763bb63a999.png)

For the future:
-We may want to make all the cards on the grid display the same height (this also needs to be done for news, events, and person)

## Related Issue
Closes #573 

## How Has This Been Tested?
Locally and in Probo

To test:
-Create at least one term in the Page Category taxonomy
-Add a few test pages and apply a category using the category side panel on the edit form
-Create a page and a view paragraph using the AZ Pages view (test both row and card)
-Add the argument for one of your taxonomy terms
-Look at your test page and make sure things display as expected

## Types of changes


## Checklist:
